src/make-http-request.coffee

```diff
@@ -43,9 +43,7 @@ makeHTTPRequest = (method, url, data, headers = {}, modify) ->
       if error?.status is 408
         resolve makeHTTPRequest.apply null, originalArguments
       else if error?
-        # Prefer rejecting with the response, since it'll have more specific information.
-        # TODO: Reject with the error as expected and access the response through `error.response` in the handler.
-        reject response ? error
+        reject error
```

Comment:
At the moment, this version of the client always errors with `'Unknown error (bad response)` in the Panoptes client, because `error` is not an instance of `Error` ie. not a JavaScript error.

I think it got lost in the noise on Slack, but I've set up a copy of PFE using these branches of the client libs on https://api-client-errors.pfe-preview.zooniverse.org/